inc/commonitiltask.class.php

```diff
@@ -1463,7 +1468,7 @@ function showForm($ID, $options = []) {
                       'value'             => $this->fields["content"],
                       'rand'              => $rand_text,
                       'editor_id'         => $content_id,
-                      'enable_fileupload' => true,
+                      'enable_fileupload' => false,
```

Comment:
Uploading files (not necessarly inlined images) during task may be a used feature, and seems to work correclty. You should not remove it.
```suggestion
```